packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.tsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * @copyright Microsoft Corporation. All rights reserved.
+ *
+ * @file Screen Reader Alert component.
+ */
+
+import * as React from 'react';
+
+import { IScreenReaderAlertProps, defaultScreenReaderAlertProps, ReadingMode } from './ScreenReaderAlert.Props';
+import './ScreenReaderAlert.scss';
+
+/**
+ * This is a screen reader alert component for developers to easily add screen reader feature to there web site.
+ * This component is a wrapper, you would like to wrap the texts for screen reader to read.
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions Live Region in MDN}
+ *
+ * All functionalities of this component is tested through:
+ * 1. JAWS 18.
+ * 2. ChromeVOX v53. (Not support ReadingMode.ReadAfterOtherContent option for ChromeVOX, it will read immediately)
+ *
+ * @example
+ * <ScreenReaderAlert>
+ *   { yourFirstAlertText }
+ *   { yourSecondAlertText }
+ * </ScreenReaderAlert>
+ */
+export class ScreenReaderAlert extends React.Component<IScreenReaderAlertProps, {}> {
+  public static defaultProps: IScreenReaderAlertProps = defaultScreenReaderAlertProps;
+
+  /**
+   * The index updated during each rendering to make React re-attach the DOM.
+   *
+   * It is critical to make React re-attach the DOM once we want it to read something. Simply updating the
+   * text context inside DOM will cause different behavior in different screen readers or browsers.
+   * Re-attaching the live region element is the most common way which is supported by most major screen reader
+   * and browser combinations.
+   */
+  private _renderIndex: number = 0;
+
+  constructor(props: IScreenReaderAlertProps) {
+    super(props);
+  }
+
+  public shouldComponentUpdate(nextProps: IScreenReaderAlertProps): boolean {
+    return this.props.readingMode !== ReadingMode.DoNotRead &&
```

Comment:
Thanks for pointing this out. Fixed by adding `role='presentation'` to the outer div to hide the text in accessibility tree.
Actually the text inside ScreenReaderAlert is hidden from visual perspective. It should also be hidden from screen reader perpective. Adding role='presentation' make screen reader skip it in DOM. E.g. JAWS users will not be able to read this text via virtual cursor.

Tested with JAWS virtual cursor, it's not reading.
However, ChromeVOX seems doesn't have a way to read like JAWS virtual cursor.